spring-boot-samples/spring-boot-sample-atmosphere/pom.xml

```diff
@@ -5,7 +5,7 @@
 		<!-- Your own application should inherit from spring-boot-starter-parent -->
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-samples</artifactId>
```

Comment:
Hello, I had to change *artifactId* value to `spring-boot-starter-parent` in order to successfully compile it.